base/applications/notepad/lang/pt-PT.rc

```diff
@@ -46,26 +46,26 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Localizar...", CMD_SEARCH
         MENUITEM "Localizar &próxima\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Substituir...\tCtrl+H", CMD_REPLACE
-        MENUITEM "Ir Para...\tCtrl+G", CMD_GOTO
+        MENUITEM "Su&bstituir...\tCtrl+H", CMD_REPLACE
+        MENUITEM "&Ir Para...\tCtrl+G", CMD_GOTO
         MENUITEM SEPARATOR
         MENUITEM "Seleccionar &tudo", CMD_SELECT_ALL
-        MENUITEM "&Data/Hora\tF5", CMD_TIME_DATE
+        MENUITEM "Data/&Hora\tF5", CMD_TIME_DATE
     END
     POPUP "F&ormatar"
     BEGIN
         MENUITEM "&Quebra automática de linha", CMD_WRAP
-        MENUITEM "&Fonte...", CMD_FONT
+        MENUITEM "&Tipo de letra...", CMD_FONT
     END
     POPUP "&Visualizar"
     BEGIN
-        MENUITEM "&Barra de status", CMD_STATUSBAR
+        MENUITEM "&Barra de estado", CMD_STATUSBAR
     END
     POPUP "&Ajuda"
```

Comment:
according to your 2k3sp2-pic I would suggest:
```suggestion
    POPUP "Aj&uda"
```
i will merge your PR afterwards.

Reply:
Resolved. That suggestion is wrong for notepad. There was a picture for a moment of regedit attached here. :D

Reply:
No. &Ficheiro and &Ajuda is correct